cves/CVE-2013-3674.yml

```
@@ -64,7 +64,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A function in a file that read from a buffer did not check the size of the buffer
```

I feel like the description could be cleaned up a bit.
```suggestion
  This vulnerability was caused by a function that read from a buffer not checking the size of the buffer
```

---

Also, where is the information from the buffer coming from?